src/test/java/TrelloApiTests.java

```diff
@@ -0,0 +1,219 @@
+import static core.TrelloServiceObject.badResponseSpecification;
+import static core.TrelloServiceObject.getBoardResponse;
+import static core.TrelloServiceObject.goodResponseSpecification;
+import static core.TrelloServiceObject.requestBuilder;
+import static core.TrelloServiceObject.resourceNotFoundResponseSpecification;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.containsString;
+
+import beans.TrelloBoardResponse;
+import core.TrelloProperties;
+import core.DataProvidersBoard;
+import java.util.List;
+import org.hamcrest.Matchers;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.collections.Lists;
+
+public class TrelloApiTests extends TrelloProperties {
+
+    private static final String BOARD_NAME = "name";
+    private static final String ERROR_MESSAGE = "errorMessage";
+    private static final String INCORRECT_BACKGROUND_MESSAGE = "badBackgroundMessage";
+    private static final String MISSING_RESOURCE_MESSAGE = "missingResourceMessage";
+    private static final String PRIVATE_PERMISSION_LEVEL = "privatePermissionLevel";
+
+    private String boardId = null;
+
+    @BeforeMethod
+    public void createBoard() {
+        TrelloBoardResponse result = getBoardResponse(
+            requestBuilder()
+                .buildRequest()
+                .sendRequestToCreateBoard()
+                .then().assertThat()
+                .spec(goodResponseSpecification())
+                .extract().response());
+        boardId = result.getId();
+    }
+
+    // TEST 1
+    @Test(groups = {"groupToDeleteBoardAfterTestStep"})
+    public void checkThatDefaultBoardDescriptionIsEmpty() {
```

Comment:
Test methods names are too long

Reply:
Fixed (I tried to make these as short as possible).